aerogear/mobile-docs
# feat: initial setup (AEROGEAR-8374)

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8374

## What
See Progress

## Why
This is the first of the docs changes for the Mobile Security Service so this pr contains additional structure changes

## Verification Steps

_Steps_
1. Run the site locally using `npm run serve`
2. Navigate to the site in a browser

_Expected Result_
1. Navigation should be working
2. Page should appear as below:
![screenshot 2019-03-05 at 16 19 09](https://user-images.githubusercontent.com/6498727/53819783-7bed3980-3f62-11e9-9d7f-3185dce52da9.png)

## Checklist:


## Progress

- [x] Set up the initial structure
- [] Add details of landing page


## Additional Notes

Adding the full team for review as it would be good to get familiar with this repo and the structure. It's not required that everyone review the code but feel free to take a look.
